shuup_checkoutfi/templates/shuup/checkoutfi/wizard_form.jinja

```diff
@@ -0,0 +1,15 @@
+{% from "shuup/admin/service_providers/wizard_macros.jinja" import service_provider_section %}
+
+{% call service_provider_section(
+    identifier=form_def.name,
+    header="Checkout.fi",
+    description=gettext("Accept credit cards using <a target='_blank' href='http://checkout.fi'>Checkout.fi</a>")
```

Comment:
There is a risk the translator breaks this url, any other way you could have the url string inside translation?